src/native/fs/fs_napi.cpp

```diff
@@ -1434,8 +1582,8 @@ void
 fs_napi(Napi::Env env, Napi::Object exports)
 {
     auto exports_fs = Napi::Object::New(env);
-
-    if(const char* env_p = std::getenv("GPFS_DL_PATH")) {
+    const char* env_p = std::getenv("GPFS_DL_PATH");
+    if (env_p != NULL) {
```

Comment:
while you're at it - better rename this variable to gpfs_dl_path instead of env_p:

```suggestion
    const char* gpfs_dl_path = std::getenv("GPFS_DL_PATH");
    if (gpfs_dl_path) {
```